osu.Android.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
This line does not seem necessary.

Reply:
the build will fail if you do not specify a configuration. This happens because osu.Android.props is imported before setting Debug as the default configuration.

You will get an empty APK when you do ``msbuild /t:SignAndroidPackage`` without this change.

Reply:
so can all this be fixed by doing `/property:Configuration=Release`?

Reply:
No. The problem is because properties are checked before the set. (osu.android.props is imported before Configuration and Platform are set to their default values)

Reply:
But specifying a default at command line should work, no?